java/src/test/java/com/google/openlocationcode/EncodingTest.java

```diff
@@ -46,7 +44,7 @@ public TestData(String line) {
 
   @Before
   public void setUp() throws Exception {
-    InputStream testDataStream = new FileInputStream(getTestFile());
+    InputStream testDataStream = new FileInputStream(TestUtils.getTestFile("encoding.csv"));
```

Comment:
The spacing looks off here. Can you re-run spotless apply?

Reply:
This seems fine. After re-run there is no change.

Reply:
That's fine, I'll fix it manually later.